### ISSUE TITLE
node:tty instanceof not working as expected

### ISSUE THREAD

**Issue by sp90** [oven-sh/bun#4024]
### What version of Bun is running?

0.7.3 (canary)

### What platform is your computer?

Darwin 22.5.0 arm64 arm

### What steps can reproduce the bug?

Clone this repo:

https://github.com/sp90/bun-node-tty

Or run following code

```js
const tty_1 = require("tty");

if (process.stdout instanceof tty_1.WriteStream) {
  console.log("stdout is a TTY");
}
```

### What is the expected behavior?

stdout is a TTY

### What do you see instead?

```bash
1 | const tty_1 = require("tty");
2 | 
3 | if (process.stdout instanceof tty_1.WriteStream) {
       ^
TypeError: instanceof called on an object with an invalid prototype property.
      at [YOUR_DIR]/node-tty/index.js:4:4
      at globalThis ([YOUR_DIR]/node-tty/index.js:5:31)
```

**Comment by Electroid** [comment]
Yep, we'll need to implement this: #2081

**Comment by birkskyum** [comment]
Works with bun 0.8 - Can be closed.

**Comment by dylan-conway** [comment]
fixed in #4179
